src/Vault.sol

```diff
@@ -107,6 +99,16 @@ contract Vault is BaseVault {
         _setBaseWithdrawalFee(baseWithdrawalFee_);
     }
 
+    function setFeeModule(address feeModule_) external virtual onlyRole(FEE_MANAGER_ROLE) {
```

Comment:
is there any reason not to have these functions directly in BaseVault?

they are used there